### PR TITLE
OSX python 3 + ITK 4.11

### PR DESCRIPTION
**Pull request by cjh1**

### CODE REVIEW

**Comment by cjh1** [comment]
@mathturtle @cryos This branch is now ready for review. I am now able to run numpy operators as well a ITK operators.

File: itk-binaries/linux/make_itk_bundle.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ $cmake_path/cmake -DCMAKE_BUILD_TYPE:STRING=Release \
   -DBUILD_EXAMPLES:BOOL=OFF \
   -DBUILD_SHARED_LIBS:BOOL=ON \
   "-DCMAKE_INSTALL_PREFIX:PATH=$workdir/install" \
-  "-DNUMPY_INCLUDE_DIR:PATH=$tvsb_dir/build/install/lib/python2.7/site-packages/numpy/core/include" \
+  "-DNUMPY_INCLUDE_DIR:PATH=$tvsb_dir/build/install/lib/python3.6/site-packages/numpy/core/include" \
```
Comment (cjh1) [inline itk-binaries/linux/make_itk_bundle.sh]: I should remove this, if we are going to merge this before linux ...

**Comment by cjh1** [comment]
Actually looking at these changes we would have todo a bit of work to enable this to be merge without effecting other platforms, as it touches common files. We may just want to wait.

**Comment by mathturtle** [comment]
Yes, there are a few changes that will need to be made if we want to avoid breaking other platforms when this is merged.  I think the only changes that will affect windows are the ones in `paraview.cmake` and `tomviz.cmake`.  It is really a question of how much we want to avoid breaking the other platforms when merging these... we probably want to avoid breaking Windows, but that is almost done anyway.

**Comment by cryos** [comment]
So my take is that we cannot break macOS or Windows. I would rather not delay for Linux, and am happy to see that fixed up after.

**Pull request merged**